YOUR_ADMIN/includes/modules/edit_orders/eo_common_address_format.php

```diff
@@ -18,85 +18,85 @@
 // $address_notifier . The notification to be raised at the end of EO's standard address elements.
 //
 ?>
-<table class="table">
-    <tr>
-        <td><?php echo zen_image(DIR_WS_IMAGES . $address_icon, $address_label); ?></td>
-        <td class="eo-label"><?php echo $address_label; ?></td>
-    </tr>
+<div role="group" aria-labelledby="sr-<?php echo $address_name; ?>">
+    <table class="table">
+        <tr>
+            <td aria-hidden="true"><?php echo zen_image(DIR_WS_IMAGES . $address_icon, $address_label); ?></td>
+            <td class="eo-label" id="sr-<?php echo $address_name; ?>" role="heading" aria-level="2"><?php echo $address_label; ?></td>
+        </tr>
+        <tr>
+            <td class="eo-label"><label for="update_<?php echo $address_name; ?>_name"><?php echo ENTRY_CUSTOMER_NAME; ?></label>:&nbsp;</td>
```

Comment:
Also for 'learning', it's important for screen-readers to ensure that there's a `label` tag associated with each input?